framework/src/ble_manager/ble_manager_state.c

```diff
@@ -190,15 +266,151 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 		ret = ble_drv_conn_is_active(con_handle, is_active);
 	} break;
 
-	case BLE_EVT_CMD_CONN_IS_ANY_ACTIVE: {
+	case BLE_CMD_CONN_IS_ANY_ACTIVE: {
 		BLE_STATE_CHECK;
 
 		bool *is_active = (bool *)msg->param;
 		ret = ble_drv_conn_is_any_active(is_active);
 	} break;
 
+	// Scanner
+	case BLE_CMD_START_SCAN: {
+		BLE_STATE_CHECK;
+
+		if (g_scan_ctx.state == BLE_SCAN_STARTED) {
+			break;
```

Comment:
I think we'd better set the `ret` as success.

Reply:
I will fix it